frontend/src/components/RosalutionHeader.vue

```diff
@@ -4,6 +4,9 @@
       <img src="@/assets/rosalution-logo.svg" class="rosalution-logo">
     </router-link>
     <div data-test="primary-content" class="content">
+      <a v-if="this.workflow_status && this.workflow_status !== 'none'" class="status-icon" data-test="status-icon">
```

Comment:
Since there is a default value, the first check to see if the `this.workflow_status` is truthy, it will always evaluate to be true.  So that first check can get removed and the 'none' check can stay in.